docs/contributing/gsoc-ideas.md

```diff
@@ -588,7 +590,7 @@ Where should you publish your draft? We prefer Dropbox Paper or
 Google Docs, since those platforms allow people to look at the text
 without having to log in or download a particular app, and you can
 update the draft as you improve your idea. In either case, you should
-post the draft for feedback in chat.zulip.org.
+post the draft for feedback in [the Zulip development community](https://zulip.com/developer-community/).
```

Comment:
Two things:
* I would recommend using a named link in this file since the link appears so many times.
* These lines need to be wrapped at 80 characters just like the surrounding text.

Reply:
Done! :)